sourcegraph/cody
# Symf: Add preamble

## Description

https://sourcegraph.slack.com/archives/C05SZB829D0/p1711637157633099?thread_ts=1711634989.767619&cid=C05SZB829D0

## Test plan

1. Run query
2. Check preamble is attached
